packages/react-button/src/components/ToggleButton/useChecked.test.ts

```diff
@@ -0,0 +1,12 @@
+import { useChecked } from './useChecked';
+import { checkedBehaviorDefinition } from './checkedBehaviorDefinition';
+import { HookTestFacade, validateBehavior } from '@fluentui/a11y-testing';
+
+describe('useChecked', () => {
```

Comment:
Let's remove this file, it's not relevant anymore

Reply:
removed :)